API/src/routes/Assessment/index.js

```diff
@@ -1,8 +1,24 @@
 const { AssessmentService } = require(`../../microservices`);
 const { ResponseHandler } = require(`../../utils`);
-
 const BASE_URL = `/assessment`;
 
-module.exports = server => {
+module.exports = (server) => {
+  server.get(BASE_URL, (req, res, next) => {
+    res.send({ msg: "get" });
+    //AssessmentService
+    //ResponseHandler(req.body, {}, {});
+  });
+
+  server.post(BASE_URL, (req, res, next) => {
+    //res.send({ msg: "post" });
+    console.log("post");
+    console.log(req.body);
+    console.log(res.json(AssessmentService.passAssessment(req.body)).body);
+    next();
+  });
```

Comment:
I'm not 100% sure what's happening here.  You're logging a lot to the console.  Wouldn't your assessment creation route be in this file?

Reply:
Yeah. I need to clean up a lot of the code. A lot of it was to see if the data was getting passed properly. I need to get rid of a lot of console.logs.